<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="VolumeButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BackwardButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>backward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ForwardButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/playIcon.jpg</normaloff>:/images/playIcon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="PauseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/pauseIcon.jpg</normaloff>:/images/pauseIcon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
